src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -124,6 +124,33 @@ object SampleWdl {
     val OutputKey = "goodbye.goodbye.out"
   }
 
+  object EmptyString extends SampleWdl {
+    override def wdlSource(runtime: String = "") =
+      """
+        |task hello {
+        |  command {
+        |    echo "Hello!"
+        |  }
+        |  output {
+        |    String empty = ""
+        |  }
+        |  RUNTIME
+        |}
+        |
+        |workflow hello {
+        |  call hello
+        |  output {
+        |   hello.empty
+        |  }
+        |}
+      """.stripMargin.replaceAll("RUNTIME", runtime)
+
+    val rawInputs = Map.empty[String, Any]
+    val OutputKey = "hello.hello.empty"
+    val OutputValue = ""
+  }
+
+
```

Comment:
As I understand it, this problem only manifests on mysql so this test won't actually check anything... I don't have an answer to this, just commenting...